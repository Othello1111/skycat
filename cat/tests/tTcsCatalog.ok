Query GSC for objects at pos 03:19:48.000 +41:30:39.00, in radius 0..10:
Query returns: 10 objects, out of 10 requested
More objects ?: YES
---Result List---
Results:

id ra dec cooSystem epoch pma pmd radvel parallax cooType band mag more preview distance pa
-
{GSC0285601078} 03:19:48.230 +41:30:42.23 {J2000} 2000 {} {} {} {} {M} {V} 9.23 {} {} 0.07 39
{GSC0285600314} 03:19:44.443 +41:30:58.21 {J2000} 2000 {} {} {} {} {M} {V} 13.98 {} {} 0.74 296
{GSC0285601162} 03:19:51.151 +41:31:25.97 {J2000} 2000 {} {} {} {} {M} {V} 12.93 {} {} 0.98 37
{GSC0285601446} 03:19:55.541 +41:31:23.74 {J2000} 2000 {} {} {} {} {M} {V} 13.72 {} {} 1.6 62
{GSC0285600342} 03:19:57.262 +41:31:34.72 {J2000} 2000 {} {} {} {} {M} {V} 12.31 {} {} 1.97 62
{GSC0285600334} 03:19:48.197 +41:32:50.14 {J2000} 2000 {} {} {} {} {M} {V} 11.77 {} {} 2.19 1
{GSC0285600886} 03:19:36.175 +41:29:58.13 {J2000} 2000 {} {} {} {} {M} {V} 10.86 {} {} 2.32 253
{GSC0285600672} 03:19:35.119 +41:30:39.67 {J2000} 2000 {} {} {} {} {M} {V} 13.39 {} {} 2.41 270
{GSC0285601758} 03:19:48.557 +41:28:09.44 {J2000} 2000 {} {} {} {} {M} {V} 13.78 {} {} 2.49 178
{GSC0285601402} 03:19:52.481 +41:33:00.04 {J2000} 2000 {} {} {} {} {M} {V} 14.67 {} {} 2.5 20

test VLT style interface:

searchClosestStar for 03:19:48.000 +41:30:39.00: returned row:
{GSC0285601078} 03:19:48.230 +41:30:42.23 {J2000} 2000 {} {} {} {} {M} {V} 9.23 {} {} 0.07 39
