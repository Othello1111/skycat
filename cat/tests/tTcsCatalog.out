TcsQueryResult

# Config entry for original catalog server:
serv_type: local
long_name: test.table
short_name: test.table
url: test.table
# End config entry


# This file contains catalog information in TCS tab table format

# Column descriptions:
id_desc= Object ID
id_type= string # Object ID

ra_desc= Alpha coordinate for the target in decimal degrees
ra_units= deg
ra_type= real
ra_range= 0.,360

dec_desc= Delta coordinate for the target in decimal degrees
dec_unit= deg
dec_type= real
dec_range= 0.,360.

cooSystem_desc= Equinox system and equinox (only 1950 or 2000 are accepted)
cooSystem_type= string
cooSystem_range= enum B1950, J2000
cooSystem_def_val= "J2000"

epoch_desc= Epoch expressed as decimal year.
epoch_type= real
epoch_range= -2000.,3000.
epoch_def_val= 2000.

pma_desc= Proper motion alpha in radians/year (-10.0 to 10.0)
pma_unit= arcsecs/year
pma_type= real
pma_range= -10.,10.
pma_def_val= 0.0

pmd_desc= Proper motion delta in radians/year (-10.0 to 10.0) 
pmd_unit= arcsecs/year
pmd_type= real
pmd_range= -10.,10.
pmd_def_val= 0.0

radvel_desc= Radial velocity in km/sec (-200000 to 200000)
radvel_unit= km/sec
radvel_type= real
radvel_range= -200000.,200000.
radvel_def_val= 0.

parallax_desc= Parallax in arcseconds (-10000 to 10000)
parallax_unit= arcseconds
parallax_type= real
parallax_range= -10000.0,10000.0
parallax_def_val= 0.0

cooType_desc= Coordinate type as "m" for mean or "a" for apparent character
cooType_type= string
cooType_range= enum "m","a"
cooType_def_val= "m"

band_desc= Magnitude wavelength band
band_type= string
band_def_val= "v"

mag_desc= Object's magnitude in given band
mag_unit= magnitude
mag_type= real
mag_def_val= 0.0

more_desc= An HTTP URL pointing to more info on the object
more_unit= http url
more_type=string
more_def_val= ""

preview_desc= An HTTP URL pointing to an image of the object
preview_unit= http url
preview_type= string
preview_def_val= ""

distance_desc= Object distance to field center
distance_unit= arcmin
distance_type= real

pa_desc= Object position angle to field center (east of north)
pa_unit= deg
pa_type= real

# NULL values
string_null= "" # empty string
real_null= 1.e-300
int_null= 4294967294 # (2^32 - 1)

id	ra	dec	cooSystem	epoch	pma	pmd	radvel	parallax	cooType	band	mag	more	preview	distance	pa
--	--	---	---------	-----	---	---	------	--------	-------	----	---	----	-------	--------	--
GSC0285600050	49.8672	41.4414	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	13.49	(null)	(null)	5.63	222
GSC0285600098	49.8875	41.571	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	13.4	(null)	(null)	4.58	322
GSC0285600288	49.8379	41.5222	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	13.13	(null)	(null)	5.12	277
GSC0285600314	49.9352	41.5162	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	13.98	(null)	(null)	0.77	293
GSC0285600334	49.9508	41.5473	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	11.77	(null)	(null)	2.16	360
GSC0285600342	49.9886	41.5263	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	12.31	(null)	(null)	1.92	62
GSC0285600418	49.8505	41.5791	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	11.69	(null)	(null)	6.08	312
GSC0285600518	49.9493	41.4296	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	12.74	(null)	(null)	4.9	181
GSC0285600526	49.8614	41.5406	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	11.25	(null)	(null)	4.39	294
GSC0285600548	49.9628	41.4361	J2000	2000	1e-300	1e-300	1e-300	1e-300	M	V	12.09	(null)	(null)	4.54	173
